projects/driver_controls/inc/analog_io.h

```diff
@@ -1,6 +1,6 @@
 #pragma once
```

Comment:
Please document the required modules that must be initialized (`adc`, `gpio`, etc.)

Reply:
Also for the other modules as well I guess, I'm not going to comment on all of them.